### PR TITLE
Update client event properties

### PR DESCRIPTION
**Pull request by everdimension**
For metabase events:
* remove user_id
* send wallet_address
* send correct wallet_provider

### CODE REVIEW

File: src/shared/analytics/analytics.client.ts [hunk 1]
```diff
@@ -1,85 +1,107 @@
...
+  getWalletProvider,
+}: {
+  getWalletProvider: (address: string) => Promise<string>;
+}) {
+  async function createWalletProperties() {
+    const address = readCachedCurrentAddress() ?? null;
+    const wallet_provider = address
+      ? await requestWithTimeout(() => getWalletProvider(address))
+      : null;
+    return { wallet_address: address?.toLowerCase(), wallet_provider } as const;
```
Comment (zerts) [inline src/shared/analytics/analytics.client.ts:35]: We should filter all `null` values from the final object for Mixpanel. 
Should we filter wallet_provider: null here?
I don't see that we send this to mixpanel, but may be I miss something

Comment (zerts) [reply src/shared/analytics/analytics.client.ts:35]: Loos like all good!

File: src/shared/analytics/analytics.background.ts [hunk 2]
```diff
@@ -40,21 +40,36 @@ function trackAppEvents({ account }: { account: Account }) {
...
-    const getUserId = () => account.getUser()?.id;
-    return createBaseParams({ ...params, userId: getUserId() });
+  const createParams: typeof createBaseParams = (eventParams) => {
+    const params: typeof eventParams & {
+      wallet_address?: string | null;
+      wallet_provider?: string | null;
+    } = eventParams;
+
+    if (!params.wallet_address) {
+      params.wallet_address = getCurrentAddress();
```
Comment (zerts) [inline src/shared/analytics/analytics.background.ts:53]: We shouldn't send null here. I see that types for getCurrentAddress allow string | null

Comment (everdimension) [reply src/shared/analytics/analytics.background.ts:53]: We have been sending it already
I'll confirm with Anya

Comment (everdimension) [reply src/shared/analytics/analytics.background.ts:53]: For mixpanel we're getting rid of all nullish params before sending: https://github.com/zeriontech/zerion-wallet-extension/blob/8ec927fa7ecf45921219bdcad41651bf09bacf8c/src/shared/analytics/mixpanel.ts#L37-L47

**Pull request merged**